src/app/cards/components/search-page/search-page.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { Store } from '@ngrx/store';
+
+import * as fromRoot from '@/app/store';
+
+@Component({
+  selector: 'tmm-search-page',
+  templateUrl: './search-page.component.html',
+  styleUrls: ['./search-page.component.scss']
+})
+export class SearchPageComponent {
+  public searchForm: FormGroup;
+
+  constructor(
+    private store: Store<fromRoot.State>,
+    formBuilder: FormBuilder
+  ) {
+    this.searchForm = formBuilder.group({
+      search: '',
+      standard_legal: false,
```

Comment:
Using snakecase here so we can simply spread the value of the form into the query params which will then be forwarded to the backend server for searching.